stubs/controller/wb.py

```diff
@@ -0,0 +1,3 @@
+from typing import Any
+
+wb: Any  # type: ignore[misc]
```

Comment:
Please can we add some structure here rather than using `Any`?

Reply:
Since we are only useing `wb` to work around a bug I've removed the stub.